lib/usecases/environment/GetAllEnvironmentsUseCase.js

```diff
@@ -85,10 +85,17 @@ class GetAllEnvironmentsUseCase {
                 currentStatus: currentStatusExpression,
                 statusHistory,
                 runNumbers: runNumbersExpression,
+                created,
             } = filter;
 
             const filterQueryBuilder = prepareQueryBuilder();
 
+            if (created) {
+                const from = created.from != null ? created.from : 0;
+                const to = created.to != null ? created.to : Date.now();
```

Comment:
I think it should be `undefined`, but  not `null`

Reply:
Yes, codebase wise it is being use `!==` undefined but I thought it was an oversight due to the fact that this service can be called by another so in theory a null value could be passed. This check is for service check as `GetAllEnvironmentsUseCase` may be called by other services which did not validate the input.
     For example `GetLogTreeUseCase` and `GetLogsByTagUseCase` are both calling `GetAllLogsUseCase`. 

For consistency, I will move to using the convention

But for your second argument on Joi, the defined DTO schema will not allow for either `null` or `undefined`. But this check is only used in the controller. It will not prevent other services passing incorrect filter values to this service.